docs/Generators/Gemmini.rst

```diff
@@ -0,0 +1,52 @@
+Gemmini
+====================================
+
+The Gemmini project is developing a systolic-array based matrix multiplication acceleration generation for the investigation of SoC integration of such accelerators. It is inspired by recent trends in machine learning accelerators for edge and mobile SoCs.
+
+Gemmini is implemented as a RoCC accelerator with non-standard RISC-V custom instructions. The Gemmini unit uses the RoCC port of a Rocket or BOOM `tile`, and by default connects to the memory system through the System Bus (i.e., directly to the L2 cache). 
+
+To add a Gemmini unit to an SoC, you should add the ``gemmini.DefaultGemminiConfig`` config mixin to the SoC configurations. To change the configuration of the Gemmini accelerator unit, you can write a custom configuration to replace the ``DefaultGemminiConfig``, which you can view under `generators/gemmini/src/main/scala/configs.scala <https://github.com/ucb-bar/gemmini/blob/master/src/main/scala/gemmini/configs.scala>`__ to see the possible configuration parameters.
+
+Alternatively, to build our example Gemmini-equipped SoC simulator, run the following commands:
+
+.. code-block:: shell
+
+    cd sims/vcs # or "cd sims/verilator"
```

Comment:
Do you want to mention the `example` configurations?

Reply:
I reorganized that part a little bit